tests/buyProducts_test.js

```diff
@@ -46,6 +47,9 @@ Data(productLinks2).Scenario('buy product', async ({ I, productPage, checkoutPag
         console.log('VAT price is: ' + vatPrice);
         let totalPrice = await checkoutPage.getTotalPrice();
         console.log('Total price is: ' + totalPrice);
+        let usdRate = await I.convertCurrencyTo('https://bank.gov.ua/NBUStatService/v1/statdirectory/exchange?valcode=USD&json');
```

Comment:
There should be no api links in scenario, move it to helper. Create one method `convertUsdToUah()` and use it.

Reply:
Fixed.